src/App.css

```diff
@@ -5,244 +5,335 @@
+            background: linear-gradient(45deg, rgb(81, 189, 252), rgb(69, 253, 177), rgb(81, 149, 252), rgb(101, 248, 224), rgb(99, 235, 128), rgb(43, 240, 125));
+			/* background: linear-gradient(45deg, red, yellow, green, blue, yellow, purple); */
+			z-index: -1;
+			background-size: 400%;
+			animation: borderbg 40s linear infinite;
+		}
+		@keyframes borderbg {
+			0%{
+				background-position: 0 0 ;
+			}
+			50%{
+				background-position: 400% 0;
+			}
+			100%{
+				background-position: 0 0 ;	
+			}
+		}
+		.movingGradient:after{
+			filter: blur(40px);
+		}
+
+.testBackground {
+    display: flex;
+    height: 100vh;
+    justify-content: center;
+    align-items: center;
+    background-color: #000;
+    font-family: Open Sans, sans-serif;
 }
 
+
 .skins-button {
-  color: #000;
-  font-size: 4rem;
-  position: absolute;
-  top: 10%;
-  left: 10%;
+    color: rgb(0, 0, 0);
+    font-size: 4rem;
+    position: absolute;
+    top: 10%;
+    left: 10%;
 }
 
 .gameplay-button {
-  color: #000;
-  font-size: 4rem;
-  position: absolute;
-  bottom: 10%;
-  left: 45%;
+    color: rgb(0, 0, 0);
+    font-size: 4rem;
+    position: relative;
+    top: 60px;
 }
 
 .upgrades-button {
-  color: #000;
-  font-size: 4rem;
-  position: absolute;
-  top: 10%;
-  right: 10%;
+    color: rgb(0, 0, 0);
+    font-size: 4rem;
+    position: absolute;
+    top: 10%;
+    right: 10%;
 }
 
 .userInput {
-  position: absolute;
-  top: 35%;
-  left: 33%;
+    color: rgb(80, 202, 86);
+    position: absolute;
+    top: 45%;
+    width: 100%;
 }
 
 .inputText {
-  font-family: 'Quicksand', sans-serif;
-  font-size: 150%;
+    font-family: 'Quicksand', sans-serif;
+    font-size: 150%;
 }
 
 .inputBox {
-  font-family: 'Quicksand', sans-serif;
-  font-size: 150%;
-  border-radius: 15px;
+    font-family: 'Quicksand', sans-serif;
+    font-size: 150%;
+    border-radius: 15px;
 }
 
 
 .slider {
-  position: relative;
-  height: 100vh;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  /* background-color: #8FF4F4; */
+    position: relative;
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    /* background-color: #8FF4F4; */
 }
 
 .image {
-  height: 600px;
-  border-radius: 10px;
+    height: 600px;
+    border-radius: 10px;
 }
 
 .right-arrow {
-  position: absolute;
-  top: 50%;
-  right: 32px;
-  font-size: 3rem;
-  color: #000;
-  z-index: 10;
-  cursor: pointer;
-  user-select: none;
+    position: absolute;
+    top: 50%;
+    right: 32px;
+    font-size: 3rem;
+    color: #000;
+    z-index: 10;
+    cursor: pointer;
+    user-select: none;
 }
 
 .left-arrow {
-  position: absolute;
-  top: 50%;
-  left: 32px;
-  font-size: 3rem;
-  color: #000;
-  z-index: 10;
-  cursor: pointer;
-  user-select: none;
+    position: absolute;
+    top: 50%;
+    left: 32px;
+    font-size: 3rem;
+    color: #000;
+    z-index: 10;
+    cursor: pointer;
+    user-select: none;
 }
 
```

Comment:
I would recommend changing the class name from "home-button" to "back-button", which is rendered as a Circle Left Button

Reply:
Kept as home-button because the pages that use this button all link to "/", which is the home page. Future implementations that might include a home and back button might need this differentiation.